### DISCUSSION TITLE
Motion controls support?

### DISCUSSION THREAD

**Discussion by cathery** [cathery/sys-con#9] [Q&A]
There's currently no way that I know to modify motion sensor values on the switch. The shared memory that is used for reading controller data is read-only.

**Comment by Chris-Donnelly** [comment]
afaik using some motion controllers is a "main-in-the-middle" job, for a dongle. Many sensors use the console or PC to perform calculations to get readings (for instance camera-based ones such as Kinect, leap, etc). I'm looking into having a laptop read Leap Motion sensor, and send UDP packets to a GIMX network node, which would in turn send the correct packets to a console (in this case XBOX One), You can possibly use a Raspberry PI or something to reduce the hardware down, but it still needs the power to read the controllers, and translate them, and then deliver them at a low enough latency to the console
